public/css/icinga/login.less

```diff
@@ -103,6 +103,24 @@
     }
   }
 
+  form[id^='form_login-']  > div:nth-child(3) {
```

Comment:
Hey @sukhwinder33445,

Since we still support IE, I don't think it is a good idea to use [CSS Grid](https://caniuse.com/#feat=css-grid).

It is still possible to fix that with "traditional" CSS though:

* The `control-group` of the remember me checkbox must have `text-align: left`
* The `label` for the remember me checkbox must have `float: left`
* The `control-label-group` must have `display: inline-block` and `width: auto`

Then you only have to add some margins.

Because we're adding a special rule/override here, it makes sense to put those rules right before the `#login` bracket closes:

```php
...
# login {
...
  // Remember me checkbox alignment
  .control-group:nth-child(3) {
    ...

    > .control-label-group {
      ...
    }

    > label {
      ...
    }
  }
}
```